File: ch1/ch1-pg19-exer4.cpp
#include <stdio.h>
#include <math.h>

int main() {
    int n;
    const double PI = acos(-1);
    scanf("%d", &n);

    printf("The value of sin(n) is: %.3f\n", sin(PI * n / 180));
    printf("The value of cos(n) is: %.3f\n", cos(PI * n / 180));

    return 0;
}

File: ch1/ch1-pg19-exer4.js
console.log("calculate the sin and cos value according to the input n degrees");
const lineRdr = require("readline");
const cmdLineRdr = lineRdr.createInterface({
    input: process.stdin
});
cmdLineRdr.on('close', () => {
    process.exit();
});

let n = 0;

cmdLineRdr.on('line', (input) => {
    n = parseInt(input);
    console.log(`The input degree is: ${n}`);
    // console.log(`The calculated value of sin(${n}) is: ${sin(n)}`);
    console.log(`The calculated value of sin(${n}) is:`);
    cmdLineRdr.close();
})